guide_tags.json

```diff
@@ -156,11 +156,10 @@
         {
             "name": "deprecated",
             "guides": [
-                "microprofile-opentracing", "microprofile-rest-client", "microprofile-metrics"
+                "microprofile-opentracing", "microprofile-opentracing-jaeger", "microshed-testing"
```

Comment:
This is incorrect, please remove `microprofile-opentracing-jaeger` from the guides list;  the list under `guides` should only be the guides being tagged as deprecated, then the alt links should have an entry where the key is the deprecated guide, and the value is the new guide to link to; the nondeprecated/newer version should not be included in the `guides` list